### ISSUE TITLE
Does this support 3rd party emote services?

### ISSUE THREAD

**Issue by maith1** [Gikkman/Java-Twirk#50]
Does this library support 3rd party emote services like FrankerFaceZ, BetterTwitchTV, 7TV, etc.?

**Comment by Gikkman** [comment]
No, unfortunately not. Integrating with 3rd parties is not in scope of this library. 

Depending on what you need, you could potentially scan the messages for the substring you're after. But if you're after the image URL or some other metadata about the emotes, then you'd need to look for another way of solving the integration.
